index.html

```diff
@@ -416,6 +417,15 @@ <h2><dfn>getEntriesByName()</dfn> method</h2>
           </section>
         </section>
       </section>
+      <section>
+        <h2><dfn>takeRecords()</dfn> method</h2>
+        <p>The <a>takeRecords()</a> method instructs the user agent to flush  entries in <a>context object</a>'s <a>observer buffer</a> and must run these steps:</p>
```

Comment:
flush  entries -> flush entries

in <a>context object -> in the <a>context object

I'm a bit confused by this.
Is this equivalent to saying 
"and must run these steps" -> "by running these steps"?

Reply:
Ack. 

> "and must run these steps" -> "by running these steps"?

The preamble is short explainer for the method, and the algorithm are the actual instructions that the UA must execute; we use this pattern in other other places in the spec.

Reply:
I agree that "by running these steps" is clearer. We could modify those other places in a followup.